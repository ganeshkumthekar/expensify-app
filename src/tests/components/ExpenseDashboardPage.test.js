import React from 'react';
import Enzyme from 'enzyme';
import { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

Enzyme.configure({ adapter: new Adapter()});

test('should render ExpenseDashboardPage correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});