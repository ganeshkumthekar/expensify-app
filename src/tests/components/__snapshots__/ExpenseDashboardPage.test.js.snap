// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseDashboardPage correctly 1`] = `ShallowWrapper {}`;

exports[`should render NotFoundPage correctly 1`] = `ShallowWrapper {}`;
